### PR TITLE
Features/ffq auto next

### PR DESCRIPTION
**Pull request by snack-0verflow**
# Description

Automatically opens the next unanswered slot

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

https://imgur.com/a/AUbsNpN#0cK24kA

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevent
- [x] I have run `npm test`
- [ ] I have run `npm run test:i18n`

### CODE REVIEW

**Comment by abubuwe** [comment]
Some comments left in these files. If not needed please delete to prevent any confusion.

**Comment by selvach** [comment]
Is it easy to make this feature configurable? I'm a little interested in trying out the behaviour with internal testers. (Auto-collapse / expand vs all open at first).

**Comment by snack-0verflow** [comment]
> Some comments left in these files. If not needed please delete to prevent any confusion.

Comments removed.

**Comment by snack-0verflow** [comment]
> Is it easy to make this feature configurable? I'm a little interested in trying out the behaviour with internal testers. (Auto-collapse / expand vs all open at first).

Made it configureable. 

Simulator Screen Shot - iPhone 11 - 2020-07-21 at 10 38 51

File: src/features/DrawerMenu.tsx [hunk 1]
```diff
@@ -177,6 +179,18 @@ export function DrawerMenu(props: DrawerContentComponentProps) {
         />
         <MenuItem label={i18n.t('privacy-policy')} onPress={() => goToPrivacy()} />
         <MenuItem label={i18n.t('delete-my-data')} onPress={() => showDeleteAlert()} />
+        <View style={styles.iconNameRow}>
+          <HeaderText>Open all FFQ</HeaderText>
+          <Switch
+            trackColor={{ false: '#767577', true: '#81b0ff' }}
```
Comment (selvach) [inline src/features/DrawerMenu.tsx:185]: Could we extract out the colours into the themes.colors files please?

File: src/features/DrawerMenu.tsx [hunk 2]
```diff
@@ -177,6 +179,18 @@ export function DrawerMenu(props: DrawerContentComponentProps) {
...
+            onValueChange={(value) => {
+              userService.toggleOpenAllFFQ();
+              setOpenAllFFQ(userService.openAllFFQ);
+              console.log(userService.openAllFFQ);
```
Comment (selvach) [inline src/features/DrawerMenu.tsx:189]: console log!

File: src/features/DrawerMenu.tsx (hunk 1 above)
Comment (snack-0verflow) [reply src/features/DrawerMenu.tsx]: Used colors from the colors file instead.

File: src/features/DrawerMenu.tsx (hunk 2 above)
Comment (snack-0verflow) [reply src/features/DrawerMenu.tsx]: Removed

**Pull request merged**